.ci/Jenkinsfile_flaky

```diff
@@ -0,0 +1,113 @@
+#!/bin/groovy
+
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+
+// Looks like 'oss:ciGroup:1' or 'oss:firefoxSmoke'
+def JOB_PARTS = params.CI_GROUP.split(':')
```

Comment:
Unrelated to the changes here, but where does the job get converted to have a colon instead of a hyphen?

 `oss-ciGroup1` > `oss:ciGroup1`

Reply:
It's what I used for the job parameter values here: https://github.com/elastic/infra/blob/master/ci/jjb/kibana-ci/defs/kibana-pipeline/kibana-pipeline-flaky-test-runner.yml#L29